src/ml/neural_net/style_transfer/mps_style_transfer_utils.h

```diff
@@ -0,0 +1,88 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+
+@interface TCMPSConvolutionDescriptor:NSObject
```

Comment:
Is this specific to Style Transfer? If yes, then the name should reflect that. If no, then why is this in the `style_transfer` subdirectory?

Reply:
Changed to Descriptor Utils